massifs/bloblastlistresponsetags.go

```diff
@@ -0,0 +1,21 @@
+package massifs
+
+// this file exists to avoid an ugly package alias
```

Comment:
yes, we should have thought about the azblob datatrails package name a bit better, to avoid these kinds of collisions

Reply:
I like that we have this in a separate file though